airflow/timetables/simple.py

```diff
@@ -170,7 +170,9 @@ def __init__(self, assets: BaseAsset) -> None:
         super().__init__()
         self.asset_condition = assets
         if isinstance(self.asset_condition, AssetAlias):
-            self.asset_condition = AssetAliasCondition(self.asset_condition.name)
+            self.asset_condition = AssetAliasCondition(
+                name=self.asset_condition.name, group=self.asset_condition.group
+            )
```

Comment:
We should probably add some `as_xxx` functions on the classes so the logic is in one place (like `to_public`).

Reply:
sounds good!